tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1297,4 +1297,19 @@ class CompletionSuite extends BaseCompletionSuite {
     _.nonEmpty
   )
 
+  checkEdit(
+    "newline-dot",
```

Comment:
It seems  tests.pc.CompletionSuite.newline-dot_2.12.10 is breaking though due to having two items.